src/routes/tftRoutes.js

```diff
@@ -6,8 +6,8 @@ function getTFTRoutes(){
     const router = express.Router()
     router.get('/v1/summoner/:summonerName', getSummonerHandler)
     router.get('/v1/match/:count/:summonerName', getRecentMatchInfo)
-    router.get('/v1/match-list/:count/:summonerName', getAllMatchInfo)
-    //router.get('/v1/matchInfo/:summonerName', getMatchInfo)
+    router.get('/v1/match-list/:summonerName', getMatchHistoryForSummoner)
```

Comment:
Consider changing the path of this route:
```suggestion
    router.get('/v1/match/history/:summonerName', getMatchHistoryForSummoner)
```